trivia.py

```diff
@@ -359,6 +365,13 @@ def _stop(self, *args):
             self._save_game()
             self.factory.running = False
 
+    def _write_log(self, logfile, f, q):
+        '''
+        Writes broken and reported questions to log file
+        '''
+        with open(config.SAVE_DIR+logfile, 'a') as log:
```

Comment:
I tried that, but then got this error message and wasn't sure of the best way to fix it.

global name 'os' is not defined